### PR TITLE
Make IDL spec compliant again

### PR DESCRIPTION
**Pull request by kainino0x**
And use hex for all the bits definitions.

And make the generated IDL slightly cleaner (removes extra newlines and beginning and end).

### CODE REVIEW

File: spec/index.bs [hunk 1]
```diff
@@ -417,7 +419,7 @@ dictionary GPUColorStateDescriptor {
 
     required GPUBlendDescriptor alphaBlend;
     required GPUBlendDescriptor colorBlend;
-    GPUColorWriteFlags writeMask = GPUColorWriteBits.ALL;
+    GPUColorWriteFlags writeMask = 15;  // GPUColorWriteBits.ALL
```
Comment (jdashg) [inline spec/index.bs]: Ew :(

Comment (jdashg) [reply spec/index.bs]: Please at least use 0x0f or 0xf?

Comment (kainino0x) [reply spec/index.bs]: Done here and in the bits definitions too.

Comment (kainino0x) [reply spec/index.bs]: And GPUTextureUsage and all the other bits definitions.

**Pull request merged**